From e2004d203f50dc3269967f5cdf9dc96882601718 Mon Sep 17 00:00:00 2001
From: Muhammad Alfi Syahrin <malfisya.dev@hotmail.com>
Date: Thu, 28 Mar 2024 10:23:38 +0700
Subject: [PATCH] Remove deprecated positional arguments for translation merges

Meson has now removed the positional arguments for
the translation merges.  This causes a FTBFS scenario.
---
 data/meson.build | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index 8b6398f..a3b075b 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -25,7 +25,6 @@ desktop_exe = configure_file(   input : meson.project_name() + '.desktop.in',
                                 configuration : desktop_conf)
 
 i18n.merge_file (
-    'appdata',
     input: meson.project_name() + '.appdata.xml.in',
     output: meson.project_name() + '.appdata.xml',
     install: true,
@@ -34,7 +33,6 @@ i18n.merge_file (
 )
 
 i18n.merge_file (
-    'desktop',
     input: desktop_exe,
     output: meson.project_name() + '.desktop',
     install: true,
@@ -44,11 +42,10 @@ i18n.merge_file (
 )
 
 i18n.merge_file (
-    'desktop',
     input: desktop_exe,
     output: meson.project_name() + '-autostart.desktop',
     install: true,
     install_dir: '/etc/xdg/autostart',
     po_dir: join_paths(meson.source_root (), 'po'),
     type: 'desktop'
-)
\ No newline at end of file
+)
-- 
2.44.0

